pkg/lib/machine-dialogs.js

```diff
@@ -74,7 +75,8 @@
         "invalid-hostkey" : translate_and_init(invalid_hostkey_tmpl),
         "not-supported" : translate_and_init(not_supported_tmpl),
         "sync-users" : translate_and_init(sync_users_tmpl),
-        "unknown-hostkey" : translate_and_init(unknown_hosts_tmpl)
+        "unknown-hostkey" : translate_and_init(unknown_hosts_tmpl),
+        "unknown-host" : translate_and_init(unknown_hosts_tmpl)
```

Comment:
"invalid-hostkey" we knew a key for a host, but it is no longer the same. It is handled very differently from "unknown-hostkey": The user is not allowed to click through or resolve the former through the UI.

"unknown-host" is a host that Cockpit doesn't know about (yet). We can't just go connecting to random hosts without the user or code adding that host.